deployer/commands/exec/infra_components.py

```diff
@@ -82,6 +91,57 @@ def root_homes(
         }
     ]
 
+    if restore_volume_id:
+        # Create PV for volume
+        pv_name = f"restore-{restore_volume_id}"
+        pv = {
+            "apiVersion": "v1",
+            "kind": "PersistentVolume",
+            "metadata": {"name": pv_name},
+            "spec": {
+                "capacity": {"storage": restore_volume_size},
```

Comment:
I suspect storage capacity is a required field. I'll try to confirm whether that's required or not. 

Reading [this warning note](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims), looks like volume expansion is triggered only when there is a mismatch between the capacity specified in the PV and the PVC.

I will confirm what happens when the specified volume on the PV and the PVC is larger than the actual size of the disk and `ALLOWVOLUMEEXPANSION` is enabled on the storageclass.

Reply:
I did some testing and storage capacity is indeed a required field.

I also tried passing a larger size than the actual disk size as the argument to see if that triggers a resize. The PV and PVC were labeled with the wrong capacity but the underlying disk was not resized.

@yuvipanda How would you like to move forward?

Reply:
@sunu ah, if it doesn't trigger a resize I would say:

1. Hard code an intentionally wrong number here (as `df` and other commands we use don't rely on it, instead using other more linux filesystem level calls)
2. Write an inline comment explaining what is going on, and that it doesn't trigger resize

This would allow us to not have the engineer doing this need to know the filesize.